int64 ar_id
---
int64 result
